web_m2x_options/README.rst

```diff
@@ -115,13 +115,18 @@ If you disable one option, you can enable it for particular field by setting "cr
 
   Whether the field should always show "Search more..." entry or not.
 
+``web_m2x_options.field_limit_entries`` *int* (Default: odoo default value is ``40``)
```

Comment:
There shouldn't be a default value, as if not defined, the limit is set by Odoo itself in their core, so IMO you shouldn't change that default value if not defined.